module/database/UserDatabase.py

```diff
@@ -43,9 +43,11 @@ def checkAuth(db, user, password):
     @style.queue
     def addUser(db, user, password):
         salt = reduce(lambda x, y: x + y, [str(random.randint(0, 9)) for i in range(0, 5)])
-        h = sha1(salt + password)
+        if isinstance(password, str):
+            password = password.decode('utf-8')
+        h = sha1((salt + password).encode('Utf-8'))
```

Comment:
Codacy found an issue: [Use of insecure MD2, MD4, MD5, or SHA1 hash function.](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5985155)